### ISSUE TITLE
Add Test Case for Proper Checker Invocation

### ISSUE THREAD

**Issue by sco1** [python-discord/flake8-annotations#55]
As seen in #54, specific parameter(s) are required in the checker function signature in order to signal to Flake8 that the checker needs to be run on the specified input. While this pitfall [was identified previously](https://github.com/python-discord/flake8-annotations/blob/2698f927f44ef56f3eab904e7526e573ff639017/flake8_annotations/checker.py#L15-L18), a regression was introduced by #53 that prevented Flake8 from determining that this plugin needed to be run. This case is missed by our unit tests, as the way they are mocked assumes that Flake8 has invoked the checker & provided the information that was requested.

To guard against any future regressions, a test case should be developed that invokes Flake8 on an input & checks that the flake8-annotations checker is being run. From some initial mucking around, it seems like a simple stdin test case may be a good candidate for at least an initial approach.

Using a simple test case:

```py
def bar(x) -> None:
    pass
```

Flake8 should yield a `TYP001` error:

```bash
$ printf "def bar(x) -> None:\n    pass\n" | flake8 -
stdin:1:9: TYP001 Missing type annotation for function argument 'x'
```

**Comment by sco1** [comment]
Blocking this for now as there seems to be an upstream issue with Coverage 5.x causing CI to fail ([reported here](https://github.com/nedbat/coveragepy/issues/883#issuecomment-567685417)].

While this seems to be resolved by pinning Coverage below 5.0, since this is not a critical test let's wait for the issue to be triaged before deciding on the resolving action.

**Comment by sco1** [comment]
A [workaround](https://github.com/nedbat/coveragepy/issues/883#issuecomment-570093918) has been identified in the upstream issue, allowing us to unblock & add this test into the next dev branch.

The immediate issue is apparently caused by the lack of an existing `.coverage` file when attempting to append coverage information. By specifying parallel runs in a `.coveragerc` file, the issue is resolved. It is not clear why the presence of `subprocess` brings out this issue, but the workaround sufficies for now.
